tests/functional/ovndbcluster_controller_test.go

```diff
@@ -27,11 +28,152 @@ import (
+						// replicas and only the targeted one has been scaled down to 1 if we don't
+						// filter by name the result would be 4 (3+1)
+						for _, d := range listDns.Items {
+							if strings.Contains(d.Name, DnsName) {
+								targetedDnsOcurrences++
+							}
+						}
+						return targetedDnsOcurrences
+					}).Should(BeNumerically("==", 1))
+				},
+				Entry("DNS entry NB", "sb"),
+				Entry("DNS entry SB", "nb"),
+			)
+			DescribeTable("Should update DNSData IP if pod IP changes",
+				func(DNSEntryName string) {
+					var clusterName types.NamespacedName
+					var OVNSBDBClusterName types.NamespacedName
+					var cluster *ovnv1.OVNDBCluster
+					var dbs []types.NamespacedName
+
+					_ = th.CreateNetworkAttachmentDefinition(types.NamespacedName{Namespace: namespace, Name: "internalapi"})
+					dbs = CreateOVNDBClusters(namespace, map[string][]string{namespace + "/internalapi": {"10.0.0.1"}}, 3)
+					OVNSBDBClusterName = types.NamespacedName{Name: dbs[1].Name, Namespace: dbs[1].Namespace}
+					cluster = GetOVNDBCluster(OVNSBDBClusterName)
+					clusterName = OVNSBDBClusterName
+					var FullDNSEntryName = DNSEntryName + "." + cluster.Namespace + ".svc"
+
+					// Check that DNSData info has been created with correct IP (10.0.0.1)
+					Eventually(func(g Gomega) {
+						// Get DNSData
+						dnsEntry := GetDNSData(types.NamespacedName{Name: "dns-ovsdbserver-sb-0", Namespace: cluster.Namespace})
+						// DNSData contains Hosts (an array), this array will contain
+						// ovsdbserver-sb-0.ns.svc and ovsdbserver-sb.ns.svc, need to get
+						// in which index is located the one which the flavor is currently testing (DNS Cname or DNS)
+						index := -1
+						for i, element := range dnsEntry.Spec.Hosts {
+							for _, el_host := range element.Hostnames {
+								if el_host == FullDNSEntryName {
+									index = i
+								}
+							}
+						}
+						g.Expect(index).To(BeNumerically(">=", 0))
+						g.Expect(dnsEntry.Spec.Hosts[index].IP).Should(Equal("10.0.0.1"))
+					}).Should(Succeed())
+
+					// Modify pod IP section
+					pod := GetPod(types.NamespacedName{Name: "ovsdbserver-sb-0", Namespace: cluster.Namespace})
+					// Create new pod NAD and add it to POD
+					var netStatus []networkv1.NetworkStatus
+					var IPString []string
+					IPString = append(IPString, "10.0.0.10")
+					netStatus = append(
+						netStatus,
+						networkv1.NetworkStatus{
+							Name: cluster.Namespace + "/internalapi",
+							IPs:  IPString,
+						},
+					)
+					netStatusAnnotation, err := json.Marshal(netStatus)
+					Expect(err).NotTo(HaveOccurred())
+					pod.Annotations[networkv1.NetworkStatusAnnot] = string(netStatusAnnotation)
+					UpdatePod(pod)
+					// End modify pod IP section
+
+					// Call reconcile loop
+					Eventually(func(g Gomega) {
+						c := GetOVNDBCluster(clusterName)
+						// Change something just to call reconcile loop
+						*&c.Spec.ElectionTimer = 11000
```

Comment:
(No action required) If it's "something", then this value seems very specific... :) you could probably just

```
*&c.Spec.ElectionTimer++
```

Reply:
I used 11000 instead of 10001 as I think it's ms, so I was adding a second instead of a milisecond :)

Reply:
but it doesn't matter what you set it to, right? What I wanted to communicate that = 11000 assumes that it's not already of that value (which is an assumption that could change in the future), while ++ would always change the value. You could as well do += 1000, with the same effect (avoiding assumptions).

Reply:
Totally true, didn't realized that I was making an assumption, will modify the value by increment and not by assignation. Thanks
Done